amundsen_application/static/js/components/TableDetail/DetailList/DetailListItem/styles.scss

```diff
@@ -37,9 +37,11 @@
 
       .name {
         margin-right: 8px;
+        max-width: 85%;
       }
       .column-type {
         margin-top: 4px;
+        max-width: 15%;
```

Comment:
Does anything happen here with smaller screens?

Reply:
Looks as good as it can be to me:
<img src='https://user-images.githubusercontent.com/1790900/60144212-694b7a00-9776-11e9-8bbe-820bf7a7f763.png' width='50%' />